iceoryx_posh/test/integrationtests/test_client_server.cpp

```diff
@@ -35,6 +35,8 @@ using namespace iox::capro;
 using namespace iox::runtime;
 using namespace iox::roudi_env;
 
+constexpr uint64_t BIG_PAYLOAD_SIZE = std::numeric_limits<uint32_t>::max() + 105UL;
```

Comment:
(optional) I would rename the variable into `PAYLOAD_LARGER_4GB` so that it becomes clearer that we are testing here a payload that exceeds 32-bit.